source/dos.h

```diff
@@ -1637,7 +1686,20 @@ int installusersoundbank( char const* filename ) {
 }
 
 
+static void load_default_sf2() {
+    // Delay loading of built-in soundfont until first used
+    // This also allows the linker to not include the entire large sf2 file if this function is not used
+    if (!internals->audio.soundbanks[ DEFAULT_SOUNDBANK_AWE32 ].sf2) {
+        internals->audio.soundbanks[ DEFAULT_SOUNDBANK_AWE32 ].sf2 = tsf_load_memory( awe32rom, sizeof( awe32rom ) );
+        if (internals->audio.current_soundbank == DEFAULT_SOUNDBANK_NONE) {
+            setsoundbank(DEFAULT_SOUNDBANK_AWE32);
+        }
+    }
+}
+
+
 void noteon( int channel, int note, int velocity) {
+    load_default_sf2();
```

Comment:
I guess it would be best to add this call to the functions "noteoff", "allnotesoff" and "setinstrument" as well, as they all result in audio commands, and it should be safe to call them without having called noteon first

Reply:
That makes sense, fixed in 09dd04f